prestoadmin/configure_cmds.py

```diff
@@ -69,15 +75,59 @@ def deploy(rolename=None):
             abort("Invalid Argument. Possible values: coordinator, workers")
 
 
-def configuration_show(file_name, should_warn=True):
-    file_path = os.path.join(constants.REMOTE_CONF_DIR, file_name)
-    if not files.exists(file_path):
+@task
+@requires_topology
```

Comment:
I don't think this should be a task -- tasks are user-facing, and the above deploy() method is very similar to this, except for specifying the directory.